calculator/src/App.js

```diff
@@ -0,0 +1,72 @@
+import "./App.css";
+import React, { Component } from "react";
+import Buttons from "./Components/Buttons";
+import Output from "./Components/Output";
+
+class App extends Component {
+  state = {
+    result: "",
```

Comment:
I'd suggest keeping track of the operator in the state so that we can avoid using eval